cmd/process-agent/subcommands/check/check.go

```diff
@@ -146,15 +147,8 @@ func MakeCommand(globalParamsGetter func() *command.GlobalParams, name string, a
 					return workloadmeta.Params{AgentType: catalog}
 				}),
 
-				// Provide tagger module
-				taggerimpl.Module(),
 				// Tagger must be initialized after agent config has been setup
-				fx.Provide(func(c config.Component) tagger.Params {
-					if c.GetBool("process_config.remote_tagger") {
-						return tagger.NewNodeRemoteTaggerParams()
-					}
-					return tagger.NewTaggerParams()
-				}),
+				dualTaggerfx.Module(common.DualTaggerParams()),
```

Comment:
It's not clear to me that this change preserves existing behavior. Previously this would only use the remote tagger if `process_config.remote_tagger` is enabled. Now, based on the implementation of `DualTaggerParams`, it will also use the remote tagger in ECS environments. 

Conceptually this arguably makes sense because it makes the decision about whether to use the remote tagger consistent with `cmd/process-agent/command/main_common.go`, but it represents a change in behavior. Can you clarify what the intention here is?

Reply:
Hey @kkhor-datadog, great catch!

This was on oversight on my part. Would you think is worth keeping the consistency or do you think is best to change it? 

I mind tells that is good to have a consistency between the main process-agent entry point and their sub commands. Unfortunately, I do not fully know the internals of the process agent.

Reply:
After some review I see this code is only used when running process agent checks manually, so mainly for testing and debugging. So it's lower priority, but I think given the size of this PR it would be helpful to preserve the existing behavior instead of changing it, and then you can leave a TODO for us to review later and make it consistent with the main process agent entry point

Reply:
Done https://github.com/DataDog/datadog-agent/pull/30584/commits/e5111fd2b85aa976fcfe95e18a163ee696ea02fe